File: Upload Media/media.py
import requests

url = "https://api.kaiza.la/v1/media"

headers = {
    'accessToken': "",
    'Content-Type': "application/x-www-form-urlencoded",
    'cache-control': "no-cache"
    }

response = requests.request("POST", url, headers=headers)

print(response.text)

File: Upload Media/media.go
package main

import (
	"fmt"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "https://api.kaiza.la/v1/media"

	req, _ := http.NewRequest("POST", url, nil)

	req.Header.Add("accessToken", "")
	req.Header.Add("cache-control", "no-cache")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}